cpu/em_cpuhotplug.py

```diff
@@ -37,6 +37,8 @@ def setUp(self):
         if distro.detect().arch not in ['ppc64', 'ppc64le']:
             self.cancel("Only supported in powerpc system")
 
+        if not cpu. _is_smt():
```

Comment:
Seems to be an space between `cpu.` and `_is_smt()`?

Reply:
ah sorry pushed now